test/dom/bindings.mocha.js

```diff
@@ -474,12 +474,13 @@ describe('bindings', function() {
     var page = app.createPage();
     var $items = page.model.at('_page.list.items');
     $items.set(['alpha', 'beta']);
-
-    var fragment = page.getFragment('Body');
+    // if getFragment called before second set() call, bindings are evaluated
+    // multiple times, leading to suggested bug below of len() called w undefined
+    page.getFragment('Body');
     // When `items` gets set to an array of one, down from two, a possible bug is the `len` function
     // getting invoked again for the no-longer-existing second item, resulting in a thrown exception.
-    page.model.set('_page.list.items', ['omega']);
-    expect(fragment).html('<div>omega [L=5]</div>');
+    $items.set(['omega']);
+    expect(page.getFragment('Body')).html('<div>omega [L=5]</div>');
```

Comment:
The original test was intentionally only calling `getFragment` once prior to updating the model, to test that existing DOM objects get updated when the model is updated.

Calling it twice means the assertion isn't verifying that the original DOM objects are being updated, so we should keep the original test setup as-is.